Libraries/LibMedia/CMakeLists.txt

```diff
@@ -6,7 +6,6 @@ endif()
 
 set(SOURCES
     Audio/Loader.cpp
-    Audio/PlaybackStream.cpp
```

Comment:
Could you expand on the commit message with a reason why we're doing this?

Reply:
Yep explained. Essentially, Distribution and Distribution_CI builds had no audio at all before my change.